packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -46,27 +46,28 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
 
     // Data from L2 deposit transaction.
     struct DepositData {
-        uint64 depositTimestamp;
-        uint64 maxFeePct;
         uint64 depositId;
-        uint256 amount;
+        uint64 depositTimestamp;
         address l2Sender;
         address recipient;
         address l1Token;
+        uint256 amount;
+        uint64 slowRelayFeePct;
+        uint64 instantRelayFeePct;
+        uint64 quoteDeadline;
```

Comment:
Sorry, maybe I'm totally missing something, but what does this variable do? If the deposit "reverts" what does that mean and how are things rolled back on L2? If it doesn't cause things to actually revert, how does it differ from depositTimestamp?

Reply:
In the context of this contract, it does not do anything. It is used exclusively on the L2 contract to limit when a deposit is valid and by the relayer to define when to compute L1 pool utilization. It is included here for completeness to duplicate the exact state statement by the relayer. We could remove this as the OO disputes can fetch this from L2. wdyt?

Reply:
regarding how it differs from `depositTimestamp`, the `depositTimestamp` will be the `block.timestamp` of that the L2 deposit is included and the `quoteDeadline` is the limit up and to which the depositor is willing to accept a quote.